Symfony/CS/Fixer/Symfony/PhpdocParamsFixer.php

```diff
@@ -145,6 +162,14 @@ private function fixDocBlock($content)
         return implode($lines);
     }
 
+    /**
+     * Get all matches.
+     *
+     * @param string $line
+     * @param bool   $matchCommentOnly
+     *
+     * @return string[]
```

Comment:
In the code you always treat result of that function as array and it always works.
So either sth can be simpled due to not returning empty-response ever, or new tests should be added to cover that case.